ui/packages/consul-ui/app/components/consul/auth-method/index.scss

```diff
@@ -1,22 +1,76 @@
+%definition-list {
```

Comment:
This was the other thing ideally I wanted to chat to you about.

What do you think about moving this into its own CSS component folder (/app/components/definition-list/), and also maybe calling it something other than `definition-list`, maybe `tabular-dl` or something to do with what it looks like?

I know naming is just one of those things thats hard to pin down, but if it helps I'd say generally styling is about how things look, so the names should be about 'how things look', whereas HTML is about what things are, so names of things are about 'what things are'.

I guess if you are in CSS think about how things look, and if you are in HTML (where things are mostly named for you) think about what things are/intention - semantic HTML.

Reply:
💯 I've created a tabular-dl styling component